client/components/image-selector/docs/example.jsx

```diff
@@ -0,0 +1,68 @@
+/** @format */
```

Comment:
This is great stuff and a very thorough README - could you get it added in to the `devdocs` area so the example can be seen in action there?

Reply:
Sure!  I actually wanted to do this, but I have a quick question here.  Since this component also relies on the media modal styles, what's the accepted way to import these files?  Toss it into `devdocs.scss` or is there a better way to get this dependency imported?

Reply:
@timmyc Cool.  Just added to the devdocs/blocks section for testing.  I moved the component to blocks as this seems a bit more appropriate since it's not a primitive UI component.

https://github.com/Automattic/wp-calypso/pull/26297/commits/a920d11283ce1152fcefd88bbaaff27ebf496539